config/config.go

```diff
@@ -174,6 +176,10 @@ type GatewayConfig struct {
 	EtcdSnapCount    *uint64 `json:",omitempty"`
 	EtcdMaxSnapFiles *uint   `json:",omitempty"`
 	EtcdMaxWalFiles  *uint   `json:",omitempty"`
+
+	// Default reporting delay for gateway internal metrics
+	InternalMetricsReportingDelay         *string
```

Comment:
please mark explicitly as a json element like the above, also comment on what this does and how it is special